cli/browser-compatibility-table.js

```diff
@@ -0,0 +1,22 @@
+export function fixMdnURL(document) {
+  // Loop over, and mutate, all 'browser_compatibility' sections
+  document.body
+    .filter(section => section.type === "browser_compatibility")
+    .forEach(section => {
+      Object.entries(section.value.data).forEach(([key, block]) => {
+        if (!!block.__compat) {
+          block = block.__compat;
+        }
+        if (!!block.mdn_url) {
+          block.mdn_url = mutateURL(block.mdn_url);
+        }
+      });
+    });
+}
+
+function mutateURL(mdn_url) {
+  return mdn_url
+    .split("/")
+    .slice(4)
+    .join("/");
```

Comment:
I'd rather you use [`const url = require('url');`](https://nodejs.org/api/url.html) and something like this:

```javascript
const url = require("url");

const u = url.parse(
  "https://github.com/mdn/stumptown-renderer/pull/247/files?a=b#anchor"
);
console.log(u.href.split(u.hostname)[1]);
```
But perhaps it should only do that if the `u.hostname === 'developer.mozilla.org` just in case.

Reply:
By the way, if you make that `u.hostname === 'developer.mozilla.org'` check it would also mean it doesn't bother if the URL is something weird.